### ISSUE TITLE
[warning] debconf: delaying package configuration

### ISSUE THREAD

**Issue by franklinyu** [tianon/docker-brew-ubuntu-core#59]
I receive this warning

``` text
debconf: delaying package configuration, since apt-utils is not installed
```

whenever I upgrade or install any package. It does not stop any action; I didn't find any issue caused by this warning.

It seems that sometimes [`apt-utils` is required to install itself](http://superuser.com/questions/413463/catch-22-need-apt-utils-to-install-apt-utils)?

**Comment by hyperknot** [comment]
I can only confirm this issue. apt-utils is a Priority: important package, should be installed by default.

**Comment by yosifkit** [comment]
The set of installed packages is controlled by Canonical in the tar files they generate for the "minimal set of packages that can be called Ubuntu": https://partner-images.canonical.com/core/ and we only [add a few tweaks](https://github.com/tianon/docker-brew-ubuntu-core/blob/3485528d76452eff9e7d3b3f222bd21a966659a5/xenial/Dockerfile#L4-L40) to make the docker images nicer.

**Comment by hyperknot** [comment]
Is there a way to contact them / open a ticket about why is this missing from partner-images?

**Comment by tianon** [comment]
Is there any harm (barring the warning) caused by not having this package installed?  I think there might be a general "Ubuntu" bug tracker on launchpad, but I'm not sure.

http://serverfault.com/a/358950 implies that it only matters for interactive installations (ie, the installer needs to format a prompt template for display to the user), so it seems like it's really not necessary for the majority of Docker users (who install applications via a `Dockerfile`, and thus non-interactively).

**Comment by franklinyu** [comment]
> Is there any harm (barring the warning) caused by not having this package installed?

No, it has not yet stopped anything from running. Just a warning, and nothing else.

> it only matters for interactive installations

I read the answer before, but I did not know that "questions about how to be configured" is just a confirmation with size information provided (since I did not found any good online explanation for it). If that is the case, we need to open a ticket to tell Canonical to suppress the warning when `apt-get` is called with option `--assume-yes`.

I Googled for the Ubuntu bug tracker, and I found [this guide](https://help.ubuntu.com/community/ReportingBugs). I think this bug belongs to the section [Reporting non-crash hardware and desktop application bugs](https://help.ubuntu.com/community/ReportingBugs#Reporting_non-crash_hardware_and_desktop_application_bugs), but it suggests to report the bug with `ubuntu-bug`, which is (as expected) not installed in the minimal Ubuntu image; I get stuck here.
